energyplus_regressions/tests/diffs/test_table_diff.py

```diff
@@ -146,7 +166,7 @@ def test_second_file_has_table_with_different_size(self):
         )
         self.assertEqual('', response[0])  # diff status
         self.assertEqual(3, response[1])  # count_of_tables
-        self.assertEqual(0, response[2])  # big diffs
+        self.assertEqual(1, response[2])  # big diffs
```

Comment:
This test is also sufficient, really, and needed to bump it up to include the big diff.